### PR TITLE
OSDOCS-4048: Adding User Telemetry Information

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
`enterprise-4.12+`

Issue:
[OSDOCS-4048](https://issues.redhat.com/browse/OSDOCS-4048)

Link to docs preview:
- [OSD](https://56971--docspreview.netlify.app/openshift-dedicated/latest/sd_support/remote_health_monitoring/about-remote-health-monitoring.html#telemetry-user-telemetry_about-remote-health-monitoring)
- [ROSA](https://56971--docspreview.netlify.app/openshift-rosa/latest/sd_support/remote_health_monitoring/about-remote-health-monitoring.html#telemetry-user-telemetry_about-remote-health-monitoring)

QE review:
- [ ] QE has approved this change.

Additional information:
This PR adds a new subsection on user telemetry. I also ported the full OSD Support documentation to ROSA. This is now in a new folder that combines the two managed services distros: `sd_support`

### CODE REVIEW

**Comment by jerolimov** [comment]
Information is true and valid. Lgtm!

File: modules/telemetry-user-telemetry.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/about-remote-health-monitoring.adoc
+
+:_content-type: CONCEPT
+[id="telemetry-user-telemetry_{context}"]
+= User Telemetry
+
+Red Hat collects anonymized  user data from your browser. This anonymized data includes what pages, features, and resource types that the user of all clusters with enabled telemetry uses.
```
Comment (cblecker) [inline modules/telemetry-user-telemetry.adoc:9]: nit: double space
```suggestion
Red Hat collects anonymized user data from your browser. This anonymized data includes what pages, features, and resource types that the user of all clusters with enabled telemetry uses.
```

**Review by cblecker — APPROVED** [review]

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**